packages/react-query-devtools/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "module": "build/esm/index.js",
   "main": "build/cjs/packages/react-query-devtools/src/index.js",
-  "browser": "build/umd/index.production.js",
+  "browser": "build/umd/noop.production.js",
```

Comment:
Since the CJS and ESM production build entrypoints are noop's, I updated this to also follow that pattern

Reply:
we now have `index.development.js`, `index.production.js`, `noop.developtment.js` and `noop.production.js` produced. I don't think that's on purpose?

Reply:
My thought was that `build/umd/index.development.js` would be distributed via CDN (to support use-cases like https://github.com/TanStack/query/issues/3916). 

I'm not entirely sure what we want to do about the `"browser"` field here. It seems that the convention for CJS and ESM production builds is to resolve to a noop for the dev tools package? The counter-point to this would be supporting bundlers like Webpack 4, which might not know about the `"exports"` field and instead use the `"browser"` field. When I tried to use the dev tools package on a project that used an older version of Webpack, I ran into that problem and couldn't use the dev tools.

Reply:
> When I tried to use the dev tools package on a project that used an older version of Webpack, I ran into that problem and couldn't use the dev tools.

yeah me to (react-scripts v3). would be good if we could support this

Reply:
I tested the change from https://github.com/TanStack/query/pull/3924/commits/d85684bb3b6e9d6b78388c46a7936130ad019690 with `react-scripts` v3 and v4. Both versions are able to load `@tanstack/react-query-devtools` properly with that change